### ISSUE TITLE
Banned games should have cards set to unmarketable automatically

### ISSUE THREAD

**Issue by bartervg** [bartervg/barter.vg#265]
# Describe the bug
Before the `banned` Steam-tracker status, unmarketable cards had to be manually set. This thread was helpful in doing that https://steamcommunity.com/groups/card-trading-card-trades/discussions/1/133255957994868314/ 

It's been over 3 years since that discussion ended, and this should be automated now. Banned games still need to be manually set to unmarketable cards. Since it's more difficult for games to add cards, it's less of an issue, but there are 565 banned games with cards and less than half are marked as unmarketable.

## Steps to reproduce the bug
1. Go to https://barter.vg/i/24187/ and see that it is banned and has cards
2. Go to https://steamcommunity.com/market/search?category_753_Game%5B%5D=tag_app_503320&category_753_cardborder%5B%5D=tag_cardborder_0&category_753_item_class%5B%5D=tag_item_class_2&appid=753 and see that cards can no longer be bought or sold on the Community Market
3. There is no warning on Barter.vg that the cards can no longer be bought or sold

## Expected behavior
If a game has cards and it is banned, the cards should be set to unmarketable automatically. 
Unconfirmed: If a game is no longer banned, the cards should be set back to marketable automatically.

### Screenshots
![image](https://user-images.githubusercontent.com/28576132/115147269-3992f380-a028-11eb-8be1-24c77f069350.png)

### Additional context, device information (OS & Browser) if applicable 
Reported by arseny92

**Comment by bartervg** [comment]
2511 games banned by Steam, as reported by Steam-tracker and recorded by Barter.vg. 61 bans seems to have been reversed. 

Bus Driver Simulator https://barter.vg/i/47295/ was banned November 2018 and unbanned June 2019 (based on when the Steam-tracker updated). Based on [this announcement](https://store.steampowered.com/news/app/679260/view/4647108190407229721), cards were added March 2018. Based on the price history, it doesn't appear as if the trading cards were unmarketable.

Other previously banned and with trading cards
The Agony https://barter.vg/i/28794/ banned November 2018
Alter World https://barter.vg/i/10640/
Zeliria Sanctuary https://barter.vg/i/76032/
Arena: Blood on the Sand VR https://barter.vg/i/29580/
J.U.R : Japan Underground Racing https://barter.vg/i/24533/
JDM Tuner Racing https://barter.vg/i/22501/
Sig.Null https://barter.vg/i/23711/
StarFringe: Adversus https://barter.vg/i/20593/
Weapons Genius https://barter.vg/i/13266/
Zzzz-Zzzz-Zzzz https://barter.vg/i/24007/

**Comment by bartervg** [comment]
Are there banned games that have marketable cards?

**Comment by Revadike** [comment]
> Are there banned games that have marketable cards?

Yes, a list of banned games (according to steamtracker) that have marketable cards (according to barter):
```
628340,599090,485580,444290,459910,552660,444790,555700,611780,1039480,344260,508670,543320,485410,543770,494660,635010,513530,618030,587830,553180,581000,437890,539350,537860,432280,482120,464760,462030,512450,264480,576880,588100,348020,545250,491080,605910,566220,523790,535400,401560,549810,492130,575870,448350,586470,586520,567250,421030,575140,528700,590420,604160,431860,485740,497350,385630,603570,636700,491170,555620,616360,631010,546380,820770,473740,500100,335550,448260,617240,479260,1088650,575130,559200,490940,583660,513540,607700,398590,380790,503320,619520,586480,537090,341980,558810,391300,528170,598470,462200,635000,473710,360850,527490,447970,448940,612430,459100,412130,628860,594640,511340,463040,409350,555940,493760,569840,585790,574810,631530,628840,638680,578260,510220,556110,605220,604900,487270,531490,647410,599100,371210,603290,631910,521830,622820,431560,493730,574850,760080,476480,360970,400250,422100,491730,586530,524340,618120,1127190,1183970,602830,603860,568450,567270,416170,595480,406670,526390,397980,578250,515310,496870,461690,582580,607270,454040,454460,371440,581010,537360,475220,526460,698720,475430,525290,461140,562760,541100,545350,569760,608560,527460,509240,527480,431990,633880,578180,528090,608390,619730,558610,596170,463240,543790,562670,412150,521930,620530,617170,593290,496500,581470,585220,587050,601650,567170,503430,518100,332780,427610,503420,556010,463550,528080,604200,571910,448250,581640,555430,621530,451830,732110,581610,365310,1102990,543780,620240,477980,576560,603890,456980,610930,473730,625950,582280,555100,605530,355120,466790,638690,371690,602170,527550,497240,539360,414290,718090,601340,585940,608830,398070,344040,399270,509990,466480,495740,569490,542430,556120,550890,476880,514710,443680,596690,605510,498840,628400,628520,644100,605520,566990,390970,567600,656090,342740,503220,480780,567220,599990,448660,398160,873150,414070,603040,508960,587210,545360,618540,586090,471910,580990,521940,487420,591010,388120,628850,884050,513630,501900,360750,533620,588090,558620,391130,581710,403570,622530,610390,303610,860860,581800,422870,535290,533150,523230,461720,567000,458940,449440,392730,495120,566060,400240,357770,495630,555980,413510,545280,491030,474820,580970,345920,501830,463050,434780,499960,635070,564520,492830,545370,522050,501920,581430,575910,636470,556070,551670,404890,586540,588420,370100,591080,454930,587590,443940,397700,470600,375760,454150,533640,527540,493280,457440,581720,603370,432020,605500,505760,581700,633370,434010,350710,485420,495350,299460,450030,633350,486820,604810,366890,565340,508140,591950,850910,542700,1231290,998490,1292790,873290,895920
```

**Comment by bartervg** [comment]
I know it's like that in the database (and it's inaccurate), but I'm asking if there are any cards in the Steam marketplace that belong to a game that is banned. If there are no exceptions, then this becomes a much easier task.

**Comment by Revadike** [comment]
One would have to scrape the steam market for banned games to find out.

**Comment by bartervg** [comment]
I browsed banned + cards (579 items). I clicked on 20 of them at random. All card sets were not marketable. I sorted by most wishlists, and the top 10 were not marketable. 

If Valve doesn't want to send checks to the developer for game sales, it makes sense that they would not want to do so on card sales either.

Therefore, I changed the Steam-tracker update process to set all banned games to unmarketable cards. When the Steam-tracker status is removed, the cards are set to marketable. 

💎 **One-time bug bounty: 5000 gems** to discover a currently banned game that has marketable cards. The new process that these cards don't exist. Only need one example to disprove this assumption. If no one claims this, then I assume this new process is accurate. 

As to checking the Steam Market, there's a request limit. I triggered it when manually clicking.
